.travis.yml

```diff
@@ -11,10 +11,9 @@ env:
   global:
     - DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
     - CC_TEST_REPORTER_ID=025bc15a0fa9afa52d86ee24fea845cf1d363f48a466bcf2cef8ab80c29acb28
-before_install:
-  - gem install bundler:1.17.3
 before_script:
-  - bundle exec rake db:setup
+  - psql -c 'create database timeoverflow_test;' -U postgres
```

Comment:
why not use `bundle exec db:create RAILS_ENV=test`? I find it a bit more intention revealing and a bit more future-proof.

Reply:
agree, not really sold on this one :) still fighting with Travis... 

About Travis @sauloperez, I got some problems yesterday night, the travis-ci.org is shutting down (in some weeks) and moving to all projects to travis-ci.com (such a bad move Travis 😠), so now all builds have a lot delay and this make the CI feedback a pain.

I think "the good solution" is to move off from Travis, to CircleCI or GitHub-Actions. But for now, I'll migrate to travis-ci.com temporally. I tried it yesterday night (all tests are passing for me locally) but I can't, due to Organization permissions. Could you please help me with that? :))